### DISCUSSION TITLE
Lands plugin migration to GriefPrevention

### DISCUSSION THREAD

**Discussion by massacred** [GriefPrevention/GriefPrevention#2137] [General]
Hello,

Is it possible to migrate from my current Lands plugin to GriefPrevention?

**Comment by bigpresh** [comment]
Sure it is.  Someone may have already written something that will help convert the data across - have a Google - if not, you'll need to write something (or employ someone to do it for you).  I've not seen the Lands plugin's data storage, so I don't know how difficult a job it'd be, but I don't imagine it'd be too tricky.
